resources/definitions/fdmprinter.def.json

```diff
@@ -4919,6 +4919,17 @@
                     "enabled": "magic_spiralize",
                     "settable_per_mesh": false,
                     "settable_per_extruder": false
+                },
+                "relative_extrusion":
+                {
+                    "label": "Use Relative Extrusion",
+                    "description": "Use relative extrusion rather than absolute extrusion. Before, Cura always used absolute extrusion. Most printers can cope with either relative or absolute extrusion.",
```

Comment:
I think we're focusing too much on the detail.

You should just update your PR at some point with what you think is good.

As long as you don't do that I'll keep the discussion alive ;)
How about `However, it's not supported by all printers and it may produce very slight deviations in the amount of deposited material compared to absolute E-steps.`
Saying it has effect on the printed object is not specific enough in my opinion..